### DISCUSSION TITLE
Call-service API error. when trying to trigger scenes

### DISCUSSION THREAD

**Discussion by mmick08** [zachowj/node-red-contrib-home-assistant-websocket#372] [Q&A]
Hi Guys,

can you explain how to trigger scenes correctly. I always get the following error:
```Call-service API error.  Error Message: extra keys not allowed @ data['entity_id']```
[
![Capture](https://user-images.githubusercontent.com/52581125/114375821-5a210080-9b85-11eb-99fe-fa91a2ca93b8.PNG)
](url)

**Comment by zachowj** [comment]
You want to use the `turn_on` service. The `apply` service is used to apply a scene without having to define it.

**Reply by mmick08** [reply]
Works! Thank you!
